app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -28,6 +28,7 @@
 import android.text.InputType;
 import android.text.TextUtils;
 import android.text.TextWatcher;
+import android.util.AttributeSet;
```

Comment:
thanks @the-dagger , i will keep that in check next time.
Should i recreate another pull request or this will be merged ?